core/src/test/java/org/apache/spark/util/collection/unsafe/sort/UnsafeExternalSorterSuite.java

```diff
@@ -503,6 +504,41 @@ public void testGetIterator() throws Exception {
     verifyIntIterator(sorter.getIterator(279), 279, 300);
   }
 
+  @Test
+  public void testOOMDuringSpill() throws Exception {
+    final UnsafeExternalSorter sorter = newSorter();
+    // we assume that given default configuration, the size of
```

Comment:
NIT NIT NIT NIT NIT NIT This comment is pretty good, but it is kind of hard to read because of the weird white space usage. Can you just make lines of 100 characters